nextcord/guild.py

```diff
@@ -2804,6 +2804,32 @@ async def fetch_roles(self, *, cache: bool = False) -> List[Role]:
 
         return roles
 
+    async def fetch_role(self, role_id: int, /) -> Role:
+        """|coro|
+
+        Retrieves a :class:`Role` that a guild has.
+
+        .. versionadded:: 3.0
+
+        Parameters
+        ----------
+        role_id: int
+            The ID of the role to fetch.
+
+        Raises
+        ------
+        HTTPException
+            Retrieving the roles failed.
+
```

Comment:
I've tested, it actually throws an exception like this
```suggestion
        Raises
        ------
        HTTPException
            Retrieving the roles failed.
        NotFound
            The role was not found.
```